samcli/local/docker/container.py

```diff
@@ -94,6 +96,24 @@ def create(self):
             "tty": False
         }
 
+        if self._delve_path:
```

Comment:
Leaving a note here for your reference. This property will become more generic soon. When it does, this if condition being specific to golang will not make sense.